### PR TITLE
DBZ-3115 Fix test failure due to unstable json key order

### PR DESCRIPTION
**Pull request by keweishang**
https://issues.redhat.com/browse/DBZ-3115

### CODE REVIEW

File: src/test/java/io/debezium/connector/vitess/AbstractVitessConnectorTest.java [hunk 1]
```diff
@@ -472,7 +475,18 @@ else if (actualValue instanceof Struct) {
                 assertStruct((Struct) value, (Struct) actualValue);
             }
             else {
-                assertEquals("Values don't match for field '" + fieldName + "'", value, actualValue);
+                Schema schema = content.schema().field(fieldName).schema();
+                if (schema.name() != null && !schema.name().isEmpty() && schema.name().equals(Json.LOGICAL_NAME)) {
```
Comment (gunnarmorling) [inline src/test/java/io/debezium/connector/vitess/AbstractVitessConnectorTest.java:479]: Could this simply be `if(Json.LOGICAL_NAME.equals(schema.name()) {`?

**Review by gunnarmorling — APPROVED** [review]
LGTM, thx! One small suggestion inline!

File: src/test/java/io/debezium/connector/vitess/AbstractVitessConnectorTest.java (hunk 1 above)
Comment (keweishang) [reply src/test/java/io/debezium/connector/vitess/AbstractVitessConnectorTest.java]: Indeed, your `if condition` is much more simplified. Thanks. Addressed.

**Review by rgibaiev — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Thx!

**Pull request merged**